src/plugins/data/server/data_views/ui_settings_wrapper.ts

```diff
@@ -14,11 +14,11 @@ export class UiSettingsServerToCommon implements UiSettingsCommon {
   constructor(uiSettings: IUiSettingsClient) {
     this.uiSettings = uiSettings;
   }
-  get(key: string) {
+  get<T = any>(key: string): Promise<T> {
     return this.uiSettings.get(key);
   }
 
-  getAll() {
+  getAll<T = any>(): Promise<Record<string, T>> {
```

Comment:
I think `<T = any>` won't be helpful here because it might have key-value pairs with values of different types

Reply:
ah, good catch